deployment/cdk/opensearch-service-migration/options.md

```diff
@@ -95,10 +95,7 @@ These tables list all CDK context configuration values a user can specify for th
-| mskRestrictPublicAccessType      | string       | "ipv4", "ipv6", "prefixList", "securityGroupId"                                                         | Specify the type used for mskRestrictPublicAccessTo from the provided examples. **Required for mskEnablePublicEndpoints**.                                                                                                                                                                                                                         |
-| mskBrokerNodeCount               | number       | 2                                                                                                       | The number of broker nodes in the MSK cluster. **NOTE** This value must be a multiple of the number of subnets to allow even distribution e.g. For two subnets I should use 2,4,8,.. broker nodes.                                                                                                                                                 |
+| mskBrokersPerAZCount             | number       | 1                                                                                                       | The number of broker nodes per MSK Availability Zone                                                                                                                                                                                                                                                                                               |
 | mskSubnetIds                     | string array | ["subnet-123456789abcdefgh", "subnet-223456789abcdefgh"]                                                | Specify the subnet IDs of an existing VPC to place MSK brokers in. **NOTE** MSK currently requires either 2 or 3 subnets to be specified and EACH subnet should use a different Availability Zone.                                                                                                                                                 |
 | mskAZCount                       | number       | 2                                                                                                       | The number of Availability Zones for the MSK cluster to use. **NOTE** This value must be 2 or 3                                                                                                                                                                                                                                                    |
 | mskARN (Not currently available) | string       | `"arn:aws:kafka:us-east-2:12345678912:cluster/msk-cluster-test/81fbae45-5d25-44bb-aff0-108e71cc079b-7"` | Supply an existing MSK cluster ARN to use. **NOTE** As MSK is using an L1 construct this is not currently available for use                                                                                                                                                                                                                        |
```

Comment:
Seems like we should remove this parameter while we are here.

Reply:
I plan to implement this in a very small follow-up PR